projects/simulation/human_dataset_generation/README.md

```diff
@@ -0,0 +1,48 @@
+# Human-Data-Generation-Framework
+
+This folder contains the code for generating the data described in "Efficient Realistic Data Generation Framework leveraging Deep Learning-based Human Digitization"
+
+## Download and reformat the CityScapes dataset
+
+1. Download the 3D human models
+
+```
+chmod +x download_models.sh
```

Comment:
Note that this step can be skipped uploading directly the bash script with execution right on the GitHub repository.

Reply:
Execution rights are added